test/LLVMIR/rocdl.mlir

```diff
@@ -0,0 +1,29 @@
+// RUN: mlir-opt %s | FileCheck %s
+
+func @rocdl_special_regs() -> !llvm.i32 {
+  // CHECK: %0 = rocdl.workitem.id.x : !llvm.i32
```

Comment:
Don't check SSA value names directly in tests:
https://github.com/tensorflow/mlir/blob/master/g3doc/TestingGuide.md

Reply:
@River707 I was following examples depicted at:
https://github.com/tensorflow/mlir/blob/master/test/LLVMIR/nvvm.mlir

How do you recommend to revise the unit test?

Reply:
For here, it should be enough to just drop the SSA value names, e.g. "%0", from the CHECK lines. The CHECK should as constrained as possible and only focus on testing what is important. The SSA value names here are not related to what you are trying to check, i.e. the custom syntax of the ops.

Reply:
Thanks. Will revise.